enterprise/backend/test/metabase_enterprise/query_field_validation/api_test.clj

```diff
@@ -79,110 +80,167 @@
                 :when (approx= b a)]
             :found)))
 
-(defn- resp=
+(defn- qv=-report
```

Comment:
This is a lot wordier than `resp=`, but the output when it fails is 1000% easier to read. It also surfaces 500s better—for whatever reason `resp=` was swallowing HTTP errors and then barfing later (which is why I added that horrible check for `:via` and `:trace`.

Anyway, the structure is pretty easy to follow once you get used to all the not-very-DRY code.

(Any attempt to DRY it up would actually be even less clear, I think)